models/payment/src/payment/payment-draft/presets/sample-data-fashion/payment-aus2.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  LocalizedString,
+  KeyReference,
+  Money,
+} from '@commercetools-test-data/commons';
+import {
+  CustomerDraft,
+  type TCustomerDraft,
+} from '@commercetools-test-data/customer';
+import * as PaymentDraft from '../..';
+import * as PaymentMethodInfo from '../../../../payment-method-info';
+import { PaymentStatusDraft } from '../../../../payment-status';
+import { TransactionDraft } from '../../../../transaction';
+import { TPaymentDraftBuilder } from '../../../types';
+
+const customerAUS = CustomerDraft.presets.sampleDataFashion
+  .sampleUsa()
+  .build<TCustomerDraft>();
+
+const paymentAUS2 = (): TPaymentDraftBuilder =>
+  PaymentDraft.presets
+    .empty()
+    .customer(KeyReference.presets.customer().key(customerAUS.key!))
+    .amountPlanned(Money.random().centAmount(4000).currencyCode('AUD'))
+    .paymentMethodInfo(
+      PaymentMethodInfo.random()
+        .method('Gift Card')
+        .name(LocalizedString.presets.empty()['en']('Gift Card'))
+    )
+    .paymentStatus(PaymentStatusDraft.presets.empty().interfaceText('Paid'))
+    .transactions([
+      TransactionDraft.presets
+        .empty()
+        .amount(Money.random().centAmount(4000).currencyCode('AUD'))
+        .state('Pending')
+        .timestamp('2023-07-01T13:00:00.000Z')
+        .type('Authorization')
```

Comment:
Same here for `TransactionType` ([values](https://docs.commercetools.com/api/projects/payments#transactiontype))

Reply:
fixed this and above with f2fdbaa 🕺